packages/trace-viewer/src/ui/timeline.tsx

```diff
@@ -95,41 +100,34 @@ export const Timeline: React.FunctionComponent<{
   let targetBar: TimelineBar | undefined = bars.find(bar => bar.action === selectedAction);
   targetBar = hoveredBar || targetBar;
 
-  const findHoveredBarIndex = (x: number, y: number) => {
+  const findHoveredBarIndex = (x: number) => {
     const time = positionToTime(measure.width, boundaries, x);
     const time1 = positionToTime(measure.width, boundaries, x - 5);
     const time2 = positionToTime(measure.width, boundaries, x + 5);
     let index: number | undefined;
-    let yDistance: number | undefined;
     let xDistance: number | undefined;
     for (let i = 0; i < bars.length; i++) {
       const bar = bars[i];
-      const yMiddle = kBarHeight / 2 + barTop(bar);
       const left = Math.max(bar.leftTime, time1);
       const right = Math.min(bar.rightTime, time2);
       const xMiddle = (bar.leftTime + bar.rightTime) / 2;
       const xd = Math.abs(time - xMiddle);
-      const yd = Math.abs(y - yMiddle);
       if (left > right)
         continue;
       // Prefer closest yDistance (the same bar), among those prefer the closest xDistance.
-      if (index === undefined ||
-          (yd < yDistance!) ||
-          (Math.abs(yd - yDistance!) < 1e-2 && xd < xDistance!)) {
+      if (index === undefined || xd < xDistance!) {
```

Comment:
- Why this change?
- The comment does not match anymore.

Reply:
I don't think users could use it / were using it.